docs/content/concepts/layer/control/catalog/_index.md

```diff
@@ -62,12 +62,11 @@ An OSCAL catalog is organized as follows, which is based on the standard OSCAL [
 
 ## Key Concepts
 
-OSCAL catalogs define a complete set of controls. The primary use of the model is representing an entire collection of controls from an organization from which control baselines are created.
-The catalog model is also used to represent the tailored baseline resulting from a processed (resolved) profile.
+OSCAL catalogs define sets of *related*, *complementary* controls. The primary use of the catalog model is to represent a comprehensive collection of controls from which an organization can create a *baseline*, by means of control selection and tailoring. A trusted machine-readable representation of a canonical control set such as NIST SP 800-53 is a significant (and arguably indispensable) asset for any organization tasked with understanding and implementing it; the OSCAL catalog model enables this by providing a common set of rules for encoding this catalog (control set), like others, thereby promoting interoperability and leveraging commodity tools. In addition, the catalog model provides a convenient representation of the tailored selection of controls, with their tailoring in place, which results when an OSCAL **profile** (which represents a set of selections and modifications of controls from catalogs, in abstracted form) is *resolved* against the catalog(s) it imports: as a "tailored control selection" (baseline) ready for display or further processing.
```

Comment:
Suggestions:
1)"OSCAL catalog model is suitable for the representation, in a machine-readable format, of control catalogs defined as sets of ... "
2) I suggest we avoid categorizing the controls as *related* or *complementary* to not limit the use of the OSCAL model. Some controls could be *related* and others could be *complementary* but totally disjoined controls can also be represented. 
3) I suggest describing the subset of controls that can be selected as *profile* to link the information already to the OSCAL **profile** model, and add that such subsets are also referred to as *baselines*. NOTE: the term *baseline8 is only used in association with  800-53. We want to describe OSCAL to the entire world not only to the US federal community.

Reply:
I agree. I will continue to review the PR, between meetings, and will provide additional comments (when necessary). If you want to discuss my comments once I will be done, we can have a meeting.